bears/vcs/git/GitCommitBear.py

```diff
@@ -66,6 +98,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             self.err('git:', repr(stderr))
             return
 
+        body = stdout.rstrip('\n')
```

Comment:
But there's also the case where we have shortlog only, where this inherently fails.